edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/DeviceScopeApiClient.cs

```diff
@@ -133,6 +150,56 @@ async Task<ScopeResult> GetIdentitiesInScope(Uri uri)
             }
         }
 
+        async Task<ScopeResult> GetIdentitiesInTargetScopeWithRetry(Uri uri, Option<string> continuationToken)
+        {
+            try
+            {
+                return await ExecuteWithRetry(
+                    () => this.GetIdentitiesInTargetScope(uri, continuationToken),
+                    Events.RetryingGetIdentities,
+                    this.retryStrategy);
+            }
+            catch (Exception e)
+            {
+                Events.ErrorInScopeResult(e);
+                throw;
+            }
+        }
+
+        async Task<ScopeResult> GetIdentitiesInTargetScope(Uri uri, Option<string> continuationLink)
+        {
+            HttpClient client = this.proxy
+                .Map(p => new HttpClient(new HttpClientHandler { Proxy = p }, disposeHandler: true))
+                .GetOrElse(() => new HttpClient());
+            using (var msg = new HttpRequestMessage(HttpMethod.Post, uri))
+            {
+                var payload = new NestedScopeRequest()
+                {
+                    // TODO: RICHMA - pass through the AuthChain
```

Comment:
>// TODO: RICHMA - pass through the AuthChain [](start = 20, length = 44)

As I'm doing manual E2E testing, I quickly realized that I forgot to implement this...will have another iteration out soon.